docs/datasets/slovak.md

```diff
@@ -0,0 +1,509 @@
+
+- Number of few-shot examples: 4
+- Prefix prompt:
+
+  ```text
+  Nasledujú texty s pridruženými otázkami a odpoveďami.
+  ```
+
+- Base prompt template:
+
+  ```text
+  Text: {text}
+  Otázka: {question}
+  Odpoveď na maximálne 3 slová:
+  ```
+
+- Instruction-tuned prompt template:
+
+  ```text
+  Text: {text}
+
+  Odpovedzte na nasledujúcu otázku týkajúcu sa textu uvedeného vyššie maximálne 3 slovami.
+
+  Otázka: {question}
+  ```
+
+You can evaluate this dataset directly as follows:
+
+```bash
+euroeval --model <model-id> --dataset multi-wiki-qa-sk
+```
+
+## Knowledge
+
+### MMLU-sk
+
+This dataset is a machine translated version of the English [MMLU
+dataset](https://openreview.net/forum?id=d7KBjmI3GmQ) and features questions within 57
+different topics, such as elementary mathematics, US history and law. The translation to
+Slovak was done by the University of Oregon as part of [this
+paper](https://aclanthology.org/2023.emnlp-demo.28/), using GPT-3.5-turbo.
+
+The original full dataset consists of 269 / 1,410 / 13,200 samples for training,
+validation and testing, respectively. We use a 1,024 / 256 / 2,048 split for training,
+validation and testing, respectively (so 3,328 samples used in total). These splits are
+new and there can thus be some overlap between the original validation and test sets and
+our validation and test sets.
+
+Here are a few examples from the training split:
+
+```json
+{
+  "text": "V akých smeroch je prípad pre humanitárnu intervenciu, ako je uvedené v tejto kapitol... mocnými štátmi.\nd. Všetky tieto možnosti.",
+  "label": "d",
+}
+```
+
+```json
+{
+  "text": "FAKTORIÁLOVÝ ANOVA sa používa v prípade, že štúdia zahŕňa viac ako 1 VI. Aký je INTER...činok VI na rovnakej úrovni ako ostatné VI",
+  "label": "a"
+}
+```
+
+```json
+{
+  "text": "Pre ktorú z týchto dvoch situácií urobí hlavná postava (ktorá používa ja/mňa/môj) nie...ie zlé\nc. Nie zlé, zlé\nd. Nie zlé, nie zlé",
+  "label": "d",
+}
+```
+
+When evaluating generative models, we use the following setup (see the
+[methodology](/methodology) for more information on how these are used):
+
+- Number of few-shot examples: 5
+- Prefix prompt:
+
+  ```text
+  Nasledujú otázky s viacerými možnosťami (s odpoveďami).
+  ```
+
+- Base prompt template:
+
+  ```text
+  Otázka: {text}
+  Odpoveď: {label}
+  ```
+
+- Instruction-tuned prompt template:
+
+  ```text
+  Otázka: {text}
+
+  Odpovedzte na nasledujúcu otázku použitím 'a', 'b', 'c' alebo 'd', a nič iné.
+  ```
+
+You can evaluate this dataset directly as follows:
+
+```bash
+euroeval --model <model-id> --dataset mmlu-sk
+```
+
+### Winogrande
+
+This dataset was published in [this paper](https://doi.org/10.1145/3474381). The
+original full dataset consists of 47 / 1,210 samples for training and testing, and we
+use 128 of the test samples for validation, resulting in a 47 / 128 / 1,085 split for
+training, validation and testing, respectively.
```

Comment:
This sounds like it's the original one - we should mention that it's translated and filtered. Just copy from some of the other non-English Winogrande docs.